Update insert-functions.md - faq (can i connect multiple functions to… (#7251)
segmentio/segment-docs: src/connections/functions/insert-functions.md

##### Can I connect an insert function to multiple destinations?

> Yes, an insert function can be connected to multiple destinations.
> 
> ##### Can I connect multiple insert functions to one destination?
> 
> No, a destination can have only one insert function connected to it at any given time.

Reviewer: ```suggestion
Yes, you can connect an insert function to multiple destinations.

##### Can I connect multiple insert functions to one destination?

No, you can only connect one insert function to a destination.
```
